content/en/tracing/troubleshooting/dotnet_diagnostic_tool.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Using the .NET diagnostic tool for troubleshooting
+kind: documentation
+---
+
+If after installing the .NET tracer your application does not produce traces as expected, you can try running the diagnostic tool for basic troubleshooting. It should help you figure out issues with your setup (for example missing environment variables, incomplete installation, unreachable agent...).
+
+The tool can be installed in three different ways:
+
+- Using the .NET SDK by running the command:
+```
+dotnet tool install -g dd-trace
+```
+- By downloading from one of the short links:
+    * Win-x64: [https://dtdg.co/dd-trace-net-win-x64](https://dtdg.co/dd-trace-net-win-x64)
+    * Linux-x64: [https://dtdg.co/dd-trace-net-linux-x64](https://dtdg.co/dd-trace-net-linux-x64)
+    * Linux-musl-x64 (Alpine): [https://dtdg.co/dd-trace-net-linux-musl-x64](https://dtdg.co/dd-trace-net-linux-musl-x64)
+ 
+- Or by downloading [from the github release page](https://github.com/DataDog/dd-trace-dotnet/releases)
+
+## Process diagnostics 
+
+For most applications, you should use the process diagnostics. For that, you need to make sure the application is running, and get the process id (pid). 
+
+To get the pid of a process on Windows, you can open the task manager, activate the "Details" tab, and look for the "PID" column. You can also run the command `tasklist /FI "IMAGENAME eq target.exe"`where `target.exe` is the name of the process.
+
+To get the pid of a process on Linux, you can run the command `ps aux | grep target` where `target` is the name of the process.
+
+The pid is typically `1` when running in a docker container. 
```

Comment:
nit - as this info should not be the most relevant to our users, I'd put everything in one line

```suggestion
To get the pid of a process on Linux, you can run the command `ps aux | grep target` where `target` is the name of the process (the pid is typically `1` when running in a docker container).
```